### PR TITLE
fix (dashpay): fix UI and translation issues

### PR DESCRIPTION
**Pull request by HashEngineering**
## Issue being fixed or feature implemented

Focus mainly on layout file changes for this.

## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [ ] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

**Comment by HashEngineering** [comment]
Russian language source needs to be updated before review and merging.

File: wallet/res/layout/contact_request_view.xml [hunk 1]
```diff
@@ -135,22 +139,26 @@
...
             android:id="@+id/accept"
             style="@style/Button.Primary.Green"
-            android:layout_width="100dp"
+            android:layout_width="120dp"
             android:layout_height="39dp"
             android:layout_marginTop="14dp"
             android:layout_marginEnd="5dp"
             android:text="@string/contact_request_accept"
+            android:maxLines="1"
+            android:ellipsize="end"
```
Comment (HashEngineering) [inline wallet/res/layout/contact_request_view.xml:148]: make sure buttons are a single line of text and truncate with ...

File: wallet/res/layout/update_profile_error.xml [hunk 2]
```diff
@@ -29,6 +29,7 @@
         android:layout_marginTop="14dp"
         android:layout_marginEnd="20dp"
         android:gravity="center"
+        android:textAlignment="center"
```
Comment (HashEngineering) [inline wallet/res/layout/update_profile_error.xml:32]: if text is long, it would be right justified.

**Review by Syn-McJ — APPROVED** [review]
Looks good

**Pull request merged**